### PR TITLE
libnpupnp: add

### PR DESCRIPTION
**Pull request by neheb**
Let's see if this works.

### CODE REVIEW

File: mingw-w64-libnpupnp/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Contributor: Rosen Penev <rosenp@gmail.com>
+
+_realname=libnpupnp
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=6.0.5
+pkgrel=1
+pkgdesc="A C++ base UPnP library, derived from Portable UPnP, a.k.a libupnp"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang32' 'clang64' 'clangarm64')
```
Comment (Biswa96) [inline mingw-w64-libnpupnp/PKGBUILD:10]:
```suggestion
mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
```

No more 32 bit packages https://www.msys2.org/news/#2023-12-13-starting-to-drop-some-32-bit-packages

File: mingw-w64-libnpupnp/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# Contributor: Rosen Penev <rosenp@gmail.com>
+
+_realname=libnpupnp
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=6.0.5
+pkgrel=1
+pkgdesc="A C++ base UPnP library, derived from Portable UPnP, a.k.a libupnp"
```
Comment (Biswa96) [inline mingw-w64-libnpupnp/PKGBUILD:8]:
```suggestion
pkgdesc="A C++ base UPnP library, derived from Portable UPnP, a.k.a libupnp (mingw-w64)"
```

File: mingw-w64-libnpupnp/PKGBUILD (hunk 1 above)
Comment (neheb) [reply mingw-w64-libnpupnp/PKGBUILD:10]: Oh that's unfortunate. Makes sense though.

**Comment by neheb** [comment]
If 32-bit is going away, why is 32-bit CI still running?

**Comment by Biswa96** [comment]
> why is 32-bit CI still running?

There are some 32 bit packages present in repository. Those are being removed slowly.

File: mingw-w64-libnpupnp/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=6.0.5
+pkgrel=1
+pkgdesc="A C++ base UPnP library, derived from Portable UPnP, a.k.a libupnp (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url='https://www.lesbonscomptes.com/upmpdcli/'
+license=('spdx:BSD-3-Clause')
+makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
```
Comment (Biswa96) [inline mingw-w64-libnpupnp/PKGBUILD:13]: Shouldn't some of these libraries be in `depends` list? See the `runtime-dependencies` section in CI log.

Comment (neheb) [reply mingw-w64-libnpupnp/PKGBUILD:13]: Done.

**Pull request merged**